app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @NurkaAmre kindly ensure you are referencing the author and not the user as previously requested.